docs/concepts/input/focus.md

```diff
@@ -0,0 +1,149 @@
+
+| NavigationMethod | Trigger Description        |
+|:-----------------|:---------------------------|
+| Tab              | Tab key press              |
+| Pointer          | Pointer interaction        |
+| Directional      | 2D Directional (`XYFocus`) |
+| Unspecified      | Default                    |
+
+## Focus Events
+
+`InputElement`s expose `GotFocus` and `LostFocus` events. The `GotFocusEventArgs` contains the `NavigationMethod` and 
+`KeyModifiers` used to trigger the focus navigation.
+
+## Focus Pseudoclasses
+
+These pseudoclasses are helpful in creating visual indicators when styling `Control`s that are `Focusable`.
+
+| Pseudoclass    | Description                                                                    |
+|:---------------|--------------------------------------------------------------------------------|
+| :focus         | The Control has focus.                                                         |
+| :focus-within  | The Control has focus or contains a descendant that has focus.                 |
+| :focus-visible | The Control has focus and received it from Tab or Directional focus navigation |
+
+## FocusManager
+
+The `FocusManager` provides global access to focus functionality, such as retrieving the currently focused element or
+clearing focus. For additional information, see the [FocusManager docs](../services/focus-manager).
+
+## Tab Focus Navigation
+
+Focus navigation by tab occurs when the user presses tab on their keyboard. `InputElement`s with their `IsTabStop` property 
+set to `true` will be available for tab focus navigation. The `TabIndex` specifies the priority with lower numeric values being 
+navigated to first. When `TabIndex` is equal, the order of appearance within the Visual Tree is used.
+
+The `KeyboardNavigation.TabNavigation` attached property can set a `KeyboardNavigationMode` onto any `InputElement` acting as 
+a container and modify its tab navigation characteristics.
+
+| KeyboardNavigationMode | Container Item Traversal                                  |
+|:-----------------------|:----------------------------------------------------------|
+| Continue               | Continues past items and into the next container          |
+| Cycle                  | Cycles through and wraps back within its own items        |
+| Contained              | Stops at the beginning/end item                           |
+| Once                   | Container and children receive focus only once as a group |
+| None                   | Items will not be focused by tab navigation               |
+| Local                  | `TabIndex` is considered for items in local subtree only  |
+
+## Directional Focus Navigation <MinVersion version="11.1" />
+
+Focus navigation through `XYFocus` is a 2D directional scheme enabling spatial navigation from the focused control 
+towards another control to a cardinal direction: left, right, up, or down. `XYFocus.NavigationModes` is `Disabled` by 
```

Comment:
By default, it's Gamepad | Remote. Which means, it is enabled only for these types of controllers.